nixos/tests/custom-ca.nix

```
@@ -191,5 +191,5 @@ in
   firefox = { error = "Security Risk"; };
   chromium = { error = "not private"; };
   qutebrowser = { args = "-T"; error = "Certificate error"; };
-  midori = { error = "Security"; };
+  #midori = { error = "Security"; }; # time out after a couple hours
```

Ah, thank you for pinging. There's a new tab feature that interferes with the test. You can fix it by:
```suggestion
  midori = { args = "-p"; error = "Security"; };
```